api/v1alpha1/selfnoderemediationtemplate_types.go

```diff
@@ -71,15 +73,18 @@ func init() {
 }
 
 func NewRemediationTemplates() []*SelfNodeRemediationTemplate {
+	templateLabels := make(map[string]string)
+	templateLabels[labels.DefaultTemplate] = "true"
 	return []*SelfNodeRemediationTemplate{
 		{
 			ObjectMeta: metav1.ObjectMeta{
-				Name: resourceDeletionTemplateName,
+				Name:   defaultTemplateName,
+				Labels: templateLabels,
```

Comment:
just fyi, this is also possible:
Labels: map[string]string {
labels.DefaultTemplate: "",
},